selenium/src/test/java/au/org/emii/portal/utils/SeleniumUtil.java

```diff
@@ -0,0 +1,135 @@
+package au.org.emii.portal.utils;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.SSLContexts;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import javax.net.ssl.SSLContext;
+import java.io.BufferedReader;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SeleniumUtil {
+
+    private static Logger log = Logger.getLogger(SeleniumUtil.class.getName());
+
+    public static String executeCurl(String command) {
+        Process curlProc;
+        String curlResponse = null;
+        try {
+            curlProc = Runtime.getRuntime().exec(command);
+
+            DataInputStream curlIn = new DataInputStream(
+                    curlProc.getInputStream());
+
+            // read the output from the command
+            BufferedReader stdInput = new BufferedReader(new
+                    InputStreamReader(curlProc.getInputStream()));
+            while ((curlResponse = stdInput.readLine()) != null) {
+                System.out.println(curlResponse);
+            }
+
+            // read any errors from the attempted command
+            BufferedReader stdError = new BufferedReader(new
+                    InputStreamReader(curlProc.getErrorStream()));
+            while ((curlResponse = stdError.readLine()) != null) {
```

Comment:
Trying to drain the stdout and sterr of a process this way will cause tests to block waiting for the wrong output.  Far easier to use java to test links.